content/get-started/climate-data-and-other-data-on-caladapt.md

```diff
@@ -11,22 +11,29 @@ order: 2
 
 ## What climate data does Cal-Adapt provide?
 
-Cal-Adapt hosts climate change projections and related data generated for California’s Fourth Climate Change Assessment. Most of the projections on Cal-Adapt were derived by statistically downscaling the outputs of general circulation models (GCMs). Learn more about [general circulation models and downscaling](https://www.carbonbrief.org/qa-how-do-climate-models-work).
+Cal-Adapt hosts climate change projections and related data generated for California’s Fourth Climate Change Assessment. Most of the projections on Cal-Adapt were derived by statistically downscaling the outputs of general circulation models (GCMs). You can learn more about general circulation models and downscaling [here](https://www.carbonbrief.org/qa-how-do-climate-models-work).
```

Comment:
@elehmer It's generally considered a best practice to avoid using text like "here" with links. Can we make the the link text "learn more about general circulation models and downscaling" and remove "here"?

Reply:
Yeah I'm of the same opinion about "here" links but that's what Nancy wrote. I will fix this one as you suggest as that looks a lot better. Thanks.

Reply:
Cleaned this up with https://github.com/berkeley-gif/caladapt-website-2021/pull/35/commits/c90509fbfe1de41341d3fedf17249f128933a43c